File: app/src/main/java/km/company/com/androiddevkit/java/HomeActivity.java
package km.company.com.androiddevkit.java;

import android.os.Bundle;
import android.os.PersistableBundle;
import android.support.annotation.Nullable;

import km.company.com.androiddevkit.java.base.BaseActivity;
import km.company.com.androiddevkit.java.utils.Log;

public class HomeActivity extends BaseActivity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState, @Nullable PersistableBundle persistentState) {
        super.onCreate(savedInstanceState, persistentState);

        Log.log("java");

        toast("java toast");
    }
}

File: app/src/main/java/km/company/com/androiddevkit/kotlin/HomeActivity.kt
package km.company.com.androiddevkit.kotlin

import android.os.Bundle
import km.company.com.androiddevkit.R
import km.company.com.androiddevkit.kotlin.base.BaseActivity
import km.company.com.androiddevkit.kotlin.utils.log

class HomeActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)

        log("kotlin")

        toast("kotlin toast")
    }
}
